app/cdap/components/WranglerGrid/TransformationToolbar/iconStore.tsx

```diff
@@ -0,0 +1,225 @@
+      stroke="#757575"
+      strokeWidth="2"
+    />
+    <path
+      d="M3.01099 13.7969H10.6937V21.4796H4.01098C3.4587 21.4796 3.01099 21.0319 3.01099 20.4796V13.7969Z"
+      stroke="#757575"
+      strokeWidth="2"
+    />
+  </svg>
+);
+export const MathIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <rect x="3" y="2.5" width="18" height="20" rx="1" stroke="#757575" strokeWidth="2" />
+    <circle cx="8.51471" cy="13.8069" r="1.30694" fill="#757575" />
+    <circle cx="8.51471" cy="17.928" r="1.30694" fill="#757575" />
+    <rect x="7" y="5.68359" width="10.7704" height="3.11704" fill="#757575" />
+    <rect x="14.8462" y="12.5" width="2.07282" height="6.73595" rx="1.03641" fill="#757575" />
+  </svg>
+);
+export const SecurityIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12 22.5C12 22.5 20 18.5 20 12.5V5.5L12 2.5L4 5.5V12.5C4 18.5 12 22.5 12 22.5Z"
+      stroke="#757575"
+      strokeWidth="2.2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const OtherIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M13 12.5C13 11.9477 12.5523 11.5 12 11.5C11.4477 11.5 11 11.9477 11 12.5C11 13.0523 11.4477 13.5 12 13.5C12.5523 13.5 13 13.0523 13 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M6 12.5C6 11.9477 5.55228 11.5 5 11.5C4.44772 11.5 4 11.9477 4 12.5C4 13.0523 4.44772 13.5 5 13.5C5.55228 13.5 6 13.0523 6 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M20 12.5C20 11.9477 19.5523 11.5 19 11.5C18.4477 11.5 18 11.9477 18 12.5C18 13.0523 18.4477 13.5 19 13.5C19.5523 13.5 20 13.0523 20 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const GridIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12 3.5H19C19.5304 3.5 20.0391 3.71071 20.4142 4.08579C20.7893 4.46086 21 4.96957 21 5.5V19.5C21 20.0304 20.7893 20.5391 20.4142 20.9142C20.0391 21.2893 19.5304 21.5 19 21.5H12M12 3.5H5C4.46957 3.5 3.96086 3.71071 3.58579 4.08579C3.21071 4.46086 3 4.96957 3 5.5V19.5C3 20.0304 3.21071 20.5391 3.58579 20.9142C3.96086 21.2893 4.46957 21.5 5 21.5H12M12 3.5V21.5"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M3 12.492H20.7689"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const SearchIconn = (
```

Comment:
Use Mui `SearchIcon`?

Reply:
SearchIconn was declared and never used so removed it from iconStore.tsx file.